classes/classes/Appearance.as

```diff
@@ -1871,6 +1877,8 @@ package classes {
 					return "gooey";
 
 				case Skin.PLAIN:
+				case Skin.BARK:
+					return "bark";
```

Comment:
shouldn't that `return "barked";`?

Reply:
idk, I think having "bark skin" sounds right.   I guess barked would work too but it sorta sounds weird.